.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.10", 3.11, 3.12]
+        python-version: [3.10, 3.11, 3.12, 3.13]
```

Comment:
The quotes around 3.10 are required: without them it is interpreted as 3.1

Reply:
yeah trailing zero breaks unfortunately